include/nitro/broken_options/parser.ipp

```diff
@@ -39,7 +39,29 @@ namespace nitro
 {
 namespace broken_options
 {
+    broken_options::group_assigner parser::group(const std::string& group_name,
+                                                 const std::string& description)
+    {
+        if (groups_.find(group_name) == groups_.end())
+        {
+            groups_.emplace(group_name, option_group(group_name, description));
+        }
+        else if (description.size())
+        {
+            raise<parser_error>("Trying to redefine group. Name: ", group_name);
+        }
+
+        return group_assigner(group_name, *this);
```

Comment:
With the suggested changes to the `option_group` and `group_assigner` classes, this should return the entry of the groups_ member.